src/app-layout/visual-refresh-toolbar/toolbar/styles.scss

```diff
@@ -12,9 +12,10 @@
   border-block-end: 1px solid awsui.$color-border-divider-panel-bottom;
   box-sizing: border-box;
   padding-block: 0;
-  padding-inline: awsui.$space-m;
+  padding-inline: awsui.$space-m 0;
+  //right padding set in child trigger-container below for focus indicator to show correctly
   position: sticky;
-  z-index: 840;
```

Comment:
I'm curious why did you remove this z-index?

Reply:
unintentional. Did a git reset and added back